lib/rules/jsx-closing-bracket-location.js

```diff
@@ -195,7 +196,9 @@ module.exports = {
      */
     function getTokensLocations(node) {
       const sourceCode = context.getSourceCode();
+      // @ts-ignore
       const opening = sourceCode.getFirstToken(node).loc.start;
+      // @ts-ignore
```

Comment:
please use ts-expect-error, with a comment, instead of ts-ignore.

Reply:
Thanks! Fixed in [eb44ba7](https://github.com/jsx-eslint/eslint-plugin-react/pull/3731/commits/eb44ba71aba5ed0d36518b3c658b7cab5e59c14f) .